src/Ant/Modal.elm

```diff
@@ -356,13 +372,10 @@ closeIcon opts =
                                     ]
                                 ]
     in
-    toUnstyled styledHtml
-
-
-
--- TODO: add type anotation
+    styledHtml
 
 
+makeBorder : (Px -> BorderStyle (TextDecorationStyle {}) -> Color -> a) -> a
```

Comment:
Thanks 🙂

Reply:
welcome, but this one the elm lsp auto inferred for me